Gemfile

```diff
@@ -4,7 +4,6 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '3.0.0'
 gem 'redis'
 gem 'resque'
-gem 'rspec', '~> 3.0'
```

Comment:
Why did you remove it from the project?

I suppose that you already have installed the RSpec on your machine but remember about potential teammates.